tsconfig.json

```diff
@@ -1,13 +1,28 @@
 {
-    "compilerOptions": {
-        "module": "commonjs",
-        "target": "es6",
-        "noImplicitAny": true,
-        "strictNullChecks": true,
-        "noEmit": true,
-        "forceConsistentCasingInFileNames": true
-    },
-    "files": [
-        "typings/index.d.ts"
-    ]
+	"compilerOptions": {
+		"alwaysStrict": true,
+		"declaration": true,
+		"lib": ["esnext"],
+		"module": "commonjs",
+		"moduleResolution": "node",
+		"noEmitOnError": true,
+		"noImplicitAny": true,
+		"noImplicitReturns": true,
+		"noImplicitThis": true,
+		"noUnusedLocals": true,
+		"noUnusedParameters": true,
+		"outDir": "./dist",
+		"pretty": true,
+		"removeComments": false,
+		"resolveJsonModule": true,
+		"sourceMap": false,
+		"strict": true,
```

Comment:
By having `strict: true` you already enable all the ones below

https://www.typescriptlang.org/docs/handbook/compiler-options.html